test/data/jwt_keys/README.md

```diff
@@ -0,0 +1,57 @@
+Custom JWT Authentication
+
+The keys in this folder "test/data/jwt_keys" are used only for the purpose of the automatic tests.
```

Comment:
Lets add these files to the .pubignore file. We will never need to publish them right?
Lets ignore the whole directory `jwt_keys`

Reply:
Same for the flutter package

Reply:
While here we should do this for the data/realm_files dir as well.

Reply:
I deleted the whole folder jwt_keys and moved the REAMDME.md content into test\README.md.
About the realm file into data/realm_files we already have in .pubignore an exclude for the file `**/*.realm` but I will also add an exclude for the whole folder.